src/opnsense/mvc/app/models/OPNsense/Backup/NextcloudSettings.xml

```diff
@@ -9,6 +9,8 @@
         </enabled>
         <url type="TextField">
             <Required>N</Required>
+            <mask>/^https?:\/\/.*[^\/]$/</mask>
```

Comment:
```suggestion
            <mask>/^https?:\/\/+*[^\/]$/</mask>
```

"https://" was valid :)

Reply:
hmm ignore for now, it's a bit late

Reply:
should this support sub-directories or not?

Reply:
yes, nextcloud is an PHP application which can be put in any directory but it is usually put into the root directory.

Reply:
your change is perfectly valid, should have been a + for sure.